edb/lib/std/20-genericfuncs.eql

```diff
@@ -103,3 +103,108 @@ std::random() -> std::float64
 {
     FROM SQL FUNCTION 'random';
 };
+
+
+# std::min
+# --------
+
+CREATE FUNCTION
+std::min(vals: SET OF anytype) -> anytype
+{
+    FROM SQL FUNCTION 'min';
+};
+
+
+# std::max
+# --------
+
+CREATE FUNCTION
+std::max(vals: SET OF anytype) -> anytype
+{
+    FROM SQL FUNCTION 'max';
+};
+
+
+# std::all
+# --------
+
+CREATE FUNCTION
+std::all(vals: SET OF std::bool) -> std::bool
+{
+    SET initial_value := True;
+    FROM SQL FUNCTION 'bool_and';
+};
+
+
+# std::any
+# --------
+
+CREATE FUNCTION
+std::any(vals: SET OF std::bool) -> std::bool
+{
+    SET initial_value := False;
+    FROM SQL FUNCTION 'bool_or';
+};
+
+
+# std::round
+# ----------
+
+CREATE FUNCTION
+std::round(val: std::decimal) -> std::decimal
```

Comment:
@vpetrovykh this still needs to be addressed.

Reply:
```
cannot create the `std::round(val: std::decimal, d: std::int64)` function: overloading "FROM SQL FUNCTION" functions is allowed only when all functions point to the same SQL function
```
Basically, there's a version of `round` where you get to specify the decimal digits and it has to be explicitly written in SQL because it involves a downcast to `int4`. This seems to clash with `FROM SQL FUNCTION 'round';`.

Reply:
got it